addons/pos_self_order/static/src/app/pages/order_history_page/order_history_page.xml

```diff
@@ -64,20 +64,24 @@
                                                 <span t-esc="line.customer_note" class="customer_note ms-1" />
                                             </div>
                                         </div>
-                                        <span t-attf-class="card-text line_price small"
+                                        <span t-attf-class="card-text line_price"
                                             t-esc="selfOrder.formatMonetary(getPrice(line))"/>
                                     </div>
                                 </div>
-                                <div class="d-flex flex-column align-items-center px-3 pt-3 bg-white">
-                                    <div class="d-grid gap-2 w-100 w-100 pb-3">
-                                        <div class="d-flex justify-content-between text-muted">
-                                            Tax:
-                                            <span t-if="!selfOrder.priceLoading" t-esc="selfOrder.formatMonetary(order.amount_tax)"/>
-                                            <span t-else="" class="spinner-border o-self-order-spinner-custom-height"></span>
-                                        </div>
-                                        <div class="d-flex justify-content-between fw-bold">
-                                            Total:
-                                            <span t-if="!selfOrder.priceLoading" t-esc="selfOrder.formatMonetary(order.amount_total)"/>
+                                <div class="d-flex mt-2 px-3">
+                                    <div class="ms-auto border-top">
+                                        <div class="table table-sm table-borderless mb-0 py-3">
+                                            <tbody>
+                                                <tr class="text-end text-muted">
+                                                    <th class="py-0">Tax:</th>
```

Comment:
I'd use a `pt-2 pb-0` classes combination here (for the other `th` as well)

| This PR | Proposal |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/8558351/4ea841c2-482e-4bd4-b59e-30c0325a04f9) | ![image](https://github.com/odoo/odoo/assets/8558351/2421b3e0-40ad-4e2a-acad-93f8ba63f034) |

Reply:
Updated, thanks (+ replace the table div by a <table>)